Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift

```diff
@@ -224,19 +274,29 @@ class JumpBackInViewModel: FeatureFlaggable {
             completion()
         }
     }
+}
+
+// MARK: - Private: Synced tab data
+private extension JumpBackInViewModel {
 
-    private func updateRemoteTabs(completion: @escaping () -> Void) {
+    var hasAccount: Bool {
+        return profile.hasSyncableAccount() &&
+                featureFlags.isFeatureEnabled(.jumpBackInSyncedTab, checking: .buildOnly)
+    }
```

Comment:
This is slightly confusing as hasAccount is not only checking for account but is also checking feature flag.

It would be good to rename `hasAccount` to something like `hasFeatureEnabledAccount` or anything that is more descriptive. @thatswinnie thoughts?

Reply:
You are right the name was confusing. Renamed it.